desktop.css

```diff
@@ -0,0 +1,928 @@
+        color: #FFFFFF;
+        flex: none;
+        flex-grow: 0;       
+    }
+    .frame43-card-works-right-block-tags {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        gap: 12px;
+        width: 236px;
+        height: 40px;
+        flex: none;
+        order: 2;
+        flex-grow: 0;
+    }
+    .frame43-card-works-right-block-html-tag {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        padding: 10px 12px;
+        height: 40px;
+        width: 56px;
+        background: rgba(255, 255, 255, 0.24);
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+    }
+    .frame43-card-works-right-block-bootstrap-tag {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        padding: 10px 12px;
+        height: 40px;
+        width: 95px;
+        background: rgba(255, 255, 255, 0.24);
+        flex: none;
+        order: 1;
+        flex-grow: 0;
+    }
+    .frame43-card-works-right-block-ruby-tag {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        padding: 10px 12px;
+        height: 40px;
+        width: 61px;
+        background: rgba(255, 255, 255, 0.24);
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+    }
+    .works-tag-text {
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 500;
+        font-size: 15px;
+        line-height: 20px;
+        color: #fff;
+    }
+    
+    .frame43-card-works-mask-group {
+        width: 327px;
+        height: 386px;
+        position: absolute;
+    }
+    .card-works-img1 {
+        position: absolute;
+        background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.56) 84.18%),
+         url('img/Card-Img-Placeholder.svg');
+        height: 386px;
+        width: 327px;
+    }
+    .mask {
+        position: absolute;
+        background: #262626;
+        height: 100%;
+        width: 100%;
+    }
+    .two {
+        display: none;
+    }
+    /* About section */
+    #about {
+        width: 39%;
+        height: 582px;
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        padding: 130px 0px;
+        gap: 24px;
+        position: relative;
+        left: 10%;
+        top: 0px;
+    }
+    #about-me {
+        width: 566px;
+        height: 78px;
+        font-family: 'Crete Round';
+        font-style: normal;
+        font-weight: 400;
+        font-size: 72px;
+        line-height: 82px;
+        color: #F55800;
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+    }
+    #about-description {
+        width: 566px;
+        height: 148px;
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 400;
+        font-size: 20px;
+        line-height: 32px;
+        color: #F4F5F7;
+        flex: none;
+        order: 1;
+        flex-grow: 0;        
+    }
```

Comment:
The background image of the about me section is covering most of the section as well as the text and the title making it unresponsive and less appealing at certain breakpoints of the desktop section as shown below. Kind check and fix them :pray: 

### Your design 

![Screenshot from 2023-01-27 17-57-14](https://user-images.githubusercontent.com/65233860/215165108-79e0eb52-fece-4b32-8161-682f10f2fcf3.png)


### Figma design 


![Screenshot from 2023-01-27 18-13-52](https://user-images.githubusercontent.com/65233860/215165127-cfd367a9-c86d-4c74-a15e-6122fb7bf06e.png)